### ISSUE TITLE
First run error

### ISSUE THREAD

**Issue by subwolf** [Storj/core#68]
### Package Versions

Replace the values below using the output from `npm list storj`.

```
/usr/local/lib
└─┬ storj-farmer@2.2.6
  └── storj@0.6.7 
```

Replace the values below using the output from `node --version`.

```
v4.4.3
```

### Expected Behavior

Please describe the program's expected behavior. Include an example of your
usage code in the back ticks below if applicable.

Starting app for the first time, error almost immediately. On Synology NAS.
```
```

### Actual Behavior

Please describe the program's actual behavior. Please include any stack traces
or log output in the back ticks below.

```
{"level":"warn","message":"you are not bound to a public address, trying traversal strategies...","timestamp":"2016-04-29T23:15:56.973Z"}
{"level":"info","message":"you are configured to tunnel up to 3 connections","timestamp":"2016-04-29T23:15:56.981Z"}
{"level":"info","message":"node created with nodeID 43bacac17d64e436ecc2eb7b8d94b1501094dbfa","timestamp":"2016-04-29T23:15:56.998Z"}
{"level":"info","message":"requesting probe from nearest neighbor","timestamp":"2016-04-29T23:15:57.003Z"}
{"level":"info","message":"sending PROBE message to {\"protocol\":\"0.6.7\",\"address\":\"api.storj.io\",\"port\":8443,\"nodeID\":\"593844dc7f0076a1aeda9a6b9788af17e67c1052\",\"lastSeen\":1461971757007}","timestamp":"2016-04-29T23:15:57.008Z"}
{"level":"info","message":"received valid message from {\"protocol\":\"0.6.6\",\"address\":\"api.storj.io\",\"port\":8443,\"nodeID\":\"593844dc7f0076a1aeda9a6b9788af17e67c1052\",\"lastSeen\":1461971757622}","timestamp":"2016-04-29T23:15:57.623Z"}
{"level":"info","message":"requesting tunnelers from nearest neighbor","timestamp":"2016-04-29T23:15:57.802Z"}
{"level":"info","message":"sending FIND_TUNNEL message to {\"protocol\":\"0.6.7\",\"address\":\"api.storj.io\",\"port\":8443,\"nodeID\":\"593844dc7f0076a1aeda9a6b9788af17e67c1052\",\"lastSeen\":1461971757803}","timestamp":"2016-04-29T23:15:57.803Z"}
{"type":"info","message":"sent telemetry report \"Success\"","timestamp":"2016-04-29T23:15:58.106Z"}
{"level":"info","message":"received valid message from {\"protocol\":\"0.6.6\",\"address\":\"api.storj.io\",\"port\":8443,\"nodeID\":\"593844dc7f0076a1aeda9a6b9788af17e67c1052\",\"lastSeen\":1461971758164}","timestamp":"2016-04-29T23:15:58.164Z"}
{"level":"info","message":"sending OPEN_TUNNEL message to {\"protocol\":\"0.6.6\",\"address\":\"api.storj.io\",\"port\":8443,\"nodeID\":\"593844dc7f0076a1aeda9a6b9788af17e67c1052\",\"lastSeen\":1461971758287}","timestamp":"2016-04-29T23:15:58.287Z"}
{"level":"info","message":"received valid message from {\"protocol\":\"0.6.6\",\"address\":\"api.storj.io\",\"port\":8443,\"nodeID\":\"593844dc7f0076a1aeda9a6b9788af17e67c1052\",\"lastSeen\":1461971758763}","timestamp":"2016-04-29T23:15:58.763Z"}
/usr/local/lib/node_modules/storj-farmer/node_modules/storj/lib/network/index.js:494
    var local = 'http://127.0.0.1:' + self._transport._server.address().port;
                                                             ^

TypeError: Cannot read property 'address' of undefined
    at /usr/local/lib/node_modules/storj-farmer/node_modules/storj/lib/network/index.js:494:62
    at /usr/local/lib/node_modules/storj-farmer/node_modules/async/lib/async.js:789:30
    at /usr/local/lib/node_modules/storj-farmer/node_modules/async/lib/async.js:167:37
    at Object.callback (/usr/local/lib/node_modules/storj-farmer/node_modules/storj/lib/network/index.js:483:7)
    at Transport.RPC._execPendingCallback (/usr/local/lib/node_modules/storj-farmer/node_modules/storj/node_modules/kad/lib/rpc.js:280:17)
    at /usr/local/lib/node_modules/storj-farmer/node_modules/storj/node_modules/kad/lib/rpc.js:184:12
    at /usr/local/lib/node_modules/storj-farmer/node_modules/storj/node_modules/kad/lib/rpc.js:254:5
    at /usr/local/lib/node_modules/storj-farmer/node_modules/storj/node_modules/kad/node_modules/async/lib/async.js:251:17
    at /usr/local/lib/node_modules/storj-farmer/node_modules/storj/node_modules/kad/node_modules/async/lib/async.js:154:25
    at /usr/local/lib/node_modules/storj-farmer/node_modules/storj/node_modules/kad/node_modules/async/lib/async.js:248:21

```

### Steps to Reproduce

Please include the steps the reproduce the issue, numbered below. Include as
much detail as possible.

1. Start program, fill in values.
2. Behaviour as above.
3. ...

**Comment by subwolf** [comment]
Just tried using settings as mentioned by @littleskunk - nat port 4001, start tunnel port 4002, end 4003. Same result.

**Comment by gordonwritescode** [comment]
Does this *only* occur after a first run wizard start in storj/farmer?

**Comment by subwolf** [comment]
I've had it running before, the difference between those occasions is unknown. I get this error repeatedly now on subsequent runs.

**Comment by redjk** [comment]
Having the same issue, also on a Synology NAS (running in Docker).  In my case the errors began on the very first run after upgrading to farmer 2.2.6 and core 0.6.7.

**Comment by subwolf** [comment]
If I set "forward": false - in the config it seems to run but gets no connections. If I set it to true the error posted in this issue occurs. I tried connecting to ports 4000 and 4001 from a remote host, those still work.

**Comment by gordonwritescode** [comment]
This will be resolved by #74
